specification/billing/resource-manager/Microsoft.Billing/preview/2024-08-01-preview/examples/CreateOrUpdateMigrationMapping.json

```diff
@@ -0,0 +1,25 @@
+{
+  "parameters": {
+    "billingAccountName": "57151837",
+    "api-version": "2024-08-01-preview",
+    "parameters": {
+      "destinationBillingScope": "/providers/Microsoft.Billing/billingAccounts/8a1d33b5-73a1-5e7f-4b29-5afaebaba7e4:03059466-caec-4aa5-b458-2071484f7d32_2019-05-31/billingProfiles/APS5-RFWH-BG7-PGB",
+      "status": "Draft"
+    }
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "id": "/providers/Microsoft.Billing/billingAccounts/{billingAccountName}/migrations/default",
+        "name": "default",
+        "type": "Microsoft.Billing/billingAccounts/migrations/default",
+        "properties": {
+          "destinationBillingScope": "/providers/Microsoft.Billing/billingAccounts/8a1d33b5-73a1-5e7f-4b29-5afaebaba7e4:03059466-caec-4aa5-b458-2071484f7d32_2019-05-31/billingProfiles/APS5-RFWH-BG7-PGB",
+          "status": "NotStarted",
```

Comment:
Shouldn't this be "Draft" in this case as well?

Reply:
CreateOrUpdate mapping is done when client sends "Draft" status. Once mapping is created, status is moved to NotStarted. Just keep logic intact as its built around this status. It moves from NotStarted to InProgress once migration starts.
Let me know if any additional info is required.

Reply:
Kept only one status as "Draft".